analyses/infercnv-consensus-cell-type/scripts/build-normal-reference/build-reference-SCPCP000015.R

```diff
@@ -114,6 +114,9 @@ clean_sce <- function(sce) {
   reducedDim(sce, "PCA") <- NULL
   reducedDim(sce, "UMAP") <- NULL
 
+  # ensure the counts matrix is sparse
+  counts(sce) <- as(counts(sce), "dgCMatrix")
```

Comment:
Update to preferred syntax.
```suggestion
  counts(sce) <- as(counts(sce), "CsparseMatrix")
```

Reply:
thanks!